src/plugins/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod main_menu_plugin;
+pub mod map_plugin;
+pub mod player_plugin;
+pub mod pregame_plugin;
```

Comment:
Given that the module is called plugins, what do you think about renaming the files within the folder to just "main_menu.rs", "map.rs", etc?

Reply:
As long as they stay in the plugins module that should be fine. I just wanted to ensure they weren't confused with the other files of the same name. For example, with this suggested change we would have two files named player.rs; one containing the plugin systems and one with the player functions. What are your thoughts?

Reply:
Okay, so the issue at hand is the existence of two files named player.rs under different contexts, where the same name could conflict with pre-existing knowledge of functionality or purpose, if I’m following.

In that case, if the goal is to avoid naming conflicts, how about we rename the player plugin to reflect a similar form of gaming terminology: Playable Character, changing it to playable_character.rs and PlayableCharacterPlugin respectively? That way, we still capture the meaning of what it means to load in a player per say with a more concise term, while keeping the files distinct.

Reply:
Sounds fair, what do we rename map_plugin.rs to?

Reply:
Done and committed